README.md

```diff
@@ -99,6 +99,13 @@ The `"argocd"` field has a few subfields:
   - `"server"` (**Mandatory**): API Server URL for the cluster (Example: `"https://kubernetes.default.svc"`)
   - `"namespace"`: Target namespace in which to deploy the manifests from source (Example: `"my-app-namespace"`)
 
+    > Usually you want to use the same namespace for Applications and AppProjects. For this, use the field `"namespace"`.
+    > 
+    > We also allow to use different namespaces for Applications and AppProjects. In that case **don't** use namespace, use `appProjectNamespace` and `applicationNamespace` instead.
+
+  - `"appProjectNamespace"`: Target namespace in which to deploy the `AppProject` resource.
+  - `"applicationNamespace"`: Target namespace in which to deploy the `Application` resource.
```

Comment:
This sets the default namespace to deploy a resource in the application if no namespace is defined on the resource's manifests

Reply:
Updated, let me know if I got it right this time 👍